src/components/export-modal/export-modal.jsx

```diff
@@ -141,7 +157,95 @@ class ExportModal extends PureComponent {
     }
   };
 
-  renderExportButton() {
+  renderExportOptions() {
+    const { isFullCollection } = this.props;
+
+    const queryClassName = classnames({
+      [style('query')]: true,
+      [style('query-is-disabled')]: isFullCollection
+    });
+
+    const queryViewerClassName = classnames({
+      [style('query-viewer')]: true,
+      [style('query-viewer-is-disabled')]: isFullCollection
+    });
+
+    if (this.props.exportProgressStatus === QUERY) {
+      return (
+        <FormGroup controlId="export-collection-option">
+          <div className={style('radio')}>
+            <label className={queryClassName}>
+              <input type="radio"
+                aria-label="Export collection with filters radio button"
+                value="filter"
+                onChange={this.handleExportOptionSelect}
+                checked={!isFullCollection}/>
+              Export query with filters &mdash; {formatNumber(this.props.count)} results (Recommended)
+            </label>
+          </div>
+          <div className={queryViewerClassName}>
+            <QueryViewer
+              query={this.props.query}
+              disabled={isFullCollection}
+              ns={this.props.ns}
+            />
+          </div>
+          <div className={style('radio')}>
+            <label>
+              <input type="radio"
+                aria-label="Export full collection radio button"
+                value="full"
+                onChange={this.handleExportOptionSelect}
+                checked={isFullCollection}/>
+              Export Full Collection
+            </label>
+          </div>
+        </FormGroup>
+      );
+    }
+  }
+
+  renderSelectFields() {
+    if (this.props.exportProgressStatus === FIELDS ) {
+      return (
+        <ExportSelectFields/>
+      );
+    }
+  }
+
+  renderSelectOutput() {
+    if (this.props.exportProgressStatus === FILETYPE) {
+      return (
+        <ExportSelectOutput
+          count={this.props.count}
+          progress={this.props.progress}
+          status={this.props.status}
+          error={this.props.error}
+          startExport={this.props.startExport}
+          selectExportFileType={this.props.selectExportFileType}
+          selectExportFileName={this.props.selectExportFileName}
+          fileType={this.props.fileType}
+          fileName={this.props.fileName}
+          cancelExport={this.props.cancelExport}
+          exportedDocsCount={this.props.exportedDocsCount}/>
+      );
+    }
+  }
+
+  renderBackButton() {
+    if (this.props.exportProgressStatus !== QUERY) {
+      return (
+        <IconTextButton
+          text="Back"
+          clickHandler={this.handleBackButton}
+          className="btn btn-default btn-sm"
+          className={style('back-button')}
```

Comment:
how do i get classnames to have both `style('back-button')` AND the usual `btn btn-default btn-sm` ?

Reply:
I think you can do https://www.npmjs.com/package/classnames#usage

```javascript
classnames('btn', 'btn-default', 'btn-small', style('back-button'));
```